### PR TITLE
PRL-3622: Add missing welsh translations

### PR DESCRIPTION
**Pull request by hcarslaw**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-3622

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/applicant/contact-preferences/contact-post-success/content.test.ts [hunk 1]
```diff
@@ -16,10 +16,10 @@ const en = {
 };
 
 const cy = {
-  caption: `Case number - welsh #${caseNumber}`,
-  title: 'Contact preferences updated -welsh',
-  text: 'You will receive updates on the case by post. - welsh',
-  continue: 'Continue - welsh',
+  caption: `Rhif yr achos #${caseNumber}`,
+  title: 'Dewisiadau cyswllt wedi’u diweddaru',
+  text: 'Byddwch yn cael diweddariadau am yr achos drwy’r post.',
```
Comment (vivek-sekhar) [inline src/main/steps/applicant/contact-preferences/contact-post-success/content.test.ts:21]: Does it require post at the end?

File: src/main/steps/applicant/contact-preferences/contact-email/content.test.ts [hunk 2]
```diff
@@ -14,15 +14,21 @@ const en = {
...
-  caption: `Case number - welsh #${caseNumber}`,
-  title: 'Contact Preferences - welsh',
-  subTitle: 'Personal details - welsh',
-  text: 'You have decided to receive updates by email. You will still receive some information by post. - welsh',
-  continue: 'Submit - welsh',
-  warningText: 'Make sure that your contact details are up to date. -welsh',
+  caption: `Rhif yr achos #${caseNumber}`,
+  title: 'Dewisiadau Cyswllt',
+  subTitle: 'Manylion personol',
+  text: 'Rydych wedi penderfynu cael diweddariadau drwy e-bost. Byddwch yn dal i gael rhywfaint o wybodaeth drwy’r post.',
```
Comment (vivek-sekhar) [inline src/main/steps/applicant/contact-preferences/contact-email/content.test.ts:26]: Does it require post at the end?

File: src/main/steps/applicant/contact-preferences/contact-post/contest.test.ts [hunk 3]
```diff
@@ -20,15 +20,17 @@ const en = {
...
-  subTitle: 'Personal details -welsh',
-  continue: 'Submit - welsh',
+  caption: `Rhif yr achos #${caseNumber}`,
+  title: 'Dewisiadau Cyswllt',
+  subTitle: 'Manylion personol',
+  continue: 'Cyflwyno',
   textList: [
-    'You have decided to receive updates by post. -welsh',
-    'You will no longer receive updates by email. You can still access previous updates through your dashboard. -welsh',
+    'Rydych wedi penderfynu cael diweddariadau drwy’r post.',
```
Comment (vivek-sekhar) [inline src/main/steps/applicant/contact-preferences/contact-post/contest.test.ts:28]: Does it require post at the end?

File: src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsGetController.ts [hunk 4]
```diff
@@ -73,7 +73,9 @@ export const validateDataCompletion = (req: AppRequest<Partial<Case>>): void =>
     if (fieldsArray.includes(key)) {
       const value = req.session.userCase[`${key}`];
       if (typeof value === 'string' && (value === null || value === undefined || value.trim() === '')) {
-        req.session.userCase[`${key}`] = '<span class="govuk-error-message">Complete this section</span>';
+        req.session.lang === 'cy'
+          ? (req.session.userCase[`${key}`] = '<span class="govuk-error-message">Complete this section (welsh)</span>')
```
Comment (vivek-sekhar) [inline src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsGetController.ts:77]: Can you please get "Complete this section" from content.ts and not directly hardcode?

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/applicant/contact-preferences/contact-post-success/content.test.ts (hunk 1 above)
Comment (hcarslaw) [reply src/main/steps/applicant/contact-preferences/contact-post-success/content.test.ts:21]: Yes, it's in the translation given in the excel document

File: src/main/steps/applicant/contact-preferences/contact-post/contest.test.ts (hunk 3 above)
Comment (hcarslaw) [reply src/main/steps/applicant/contact-preferences/contact-post/contest.test.ts:28]: Yes, it's in the translation given in the excel document

File: src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsGetController.ts (hunk 4 above)
Comment (hcarslaw) [reply src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsGetController.ts]: Changed to use content.ts

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**